src/me/fru1t/subedit/utils/Dialogue.kt

```diff
@@ -0,0 +1,15 @@
+package me.fru1t.subedit.utils
+
+data class Dialogue(val layer: String, val start: String, val end: String, val style: String, val name: String,
```

Comment:
Consider renaming this to something like `DialogueLine` as it represents a single line (and not the entire dialogue block within the ass file)